department-of-veterans-affairs/vets-website
# Add discontinued incomes section

## Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
Import Discontinued Incomes - Section 11 into the Income & Asset Statement form (21P-0969)

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86454

## Associated Pull Request(s)
n/a

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `income_and_assets_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/income_and_assets_form_enabled
4. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969/` in your browser

## How to verify
1. Go to chapter 11 - Discontinued Incomes (/discontinued-incomes-summary)
2. Verify that an discontinued income can be added, edited, and deleted
3. Verify that discontinued incomes are displayed on the 'Review' page

## What areas of the site does it impact?
Income & Asset Statement Application

## Screenshots
### Desktop
#### Summary Page
| No Incomes      | Incomes       |
| ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_ (21)](https://github.com/user-attachments/assets/79298626-ad08-4a09-8a9e-c6c25b14747a)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (28)](https://github.com/user-attachments/assets/78b2cd4d-2909-410a-9c3c-1188f9d406c2)



### Item Pages
| Relationship | Payer | Type | 
| ----------- | ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_form-saved (15)](https://github.com/user-attachments/assets/ff8dc5f1-903a-4461-813d-fc4f857126fe)|![localhost_3001_income-and-asset-statement-form-21p-0969_form-saved (16)](https://github.com/user-attachments/assets/f81a4621-901f-4c47-9821-1839a5cb4a2c)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (24)](https://github.com/user-attachments/assets/2c651911-215a-48a0-a3e0-224f3a2b081a)|

Frequency | Date | Amount |
 ----------- | ----------- | ----------- | 
![localhost_3001_income-and-asset-statement-form-21p-0969_ (25)](https://github.com/user-attachments/assets/1f153c8d-c72c-4152-bafc-d4c281e2c5d9)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (26)](https://github.com/user-attachments/assets/8ffd10e6-0421-4e22-b576-55872cb32d6b)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (27)](https://github.com/user-attachments/assets/c8a00192-d06e-465b-ad12-41f0614507a9)|



### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
